### PR TITLE
some #150 cleanup

### PR DESCRIPTION
**Pull request by MHBauer**
updated the examples

started using loggers instead of print statements

### CODE REVIEW

File: pkg/apiserver/apiserver.go [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package apiserver
 
 import (
+	"github.com/golang/glog"
+
```
Comment (duglin) [inline pkg/apiserver/apiserver.go]: only if you're in here for some other reason... let's remove these blank lines in the imports.
I think the traditional way is to have one between golang imports and "others" but not between groups of others.
e.g.  https://github.com/kubernetes/kubernetes/blob/master/pkg/api/conversion.go

**Comment by duglin** [comment]
LGTM
with one minor request

File: cmd/service-catalog/server.go [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
 	// _ "k8s.io/kubernetes/pkg/api/install"
 )
 
 func main() {
-	// commented out until I know what this does
-	// logs.InitLogs()
-	// defer logs
+	logs.InitLogs()
+	// make sure we print all the logs while shutting down.
+	defer logs.FlushLogs()
```
Comment (martinmaly) [inline cmd/service-catalog/server.go]: will the deferred code run when line 26 (os.Exit) runs?

Comment (krancour) [inline cmd/service-catalog/server.go]: It will not. https://play.golang.org/p/NIpT8_e73L

Comment (MHBauer) [inline cmd/service-catalog/server.go]: Well, I took a guess on why that was there. No idea again. Take it out?

Comment (krancour) [inline cmd/service-catalog/server.go]: I am not familiar enough with the k8s logging pkg, but I presume that if flushing the logs is important to do before exiting 0, it's even more important to do before exiting non-zero. Can another flush just be added before `os.Exit(1)` for good measure?

**Comment by arschles** [comment]
LGTM assuming @duglin's request is addressed

File: cmd/service-catalog/server.go (hunk 2 above)
Comment (MHBauer) [inline cmd/service-catalog/server.go]: Good idea. I cribbed the defer from other apiserver implementations. I will add a direct call to flush before exit.

File: pkg/apiserver/apiserver.go (hunk 1 above)
Comment (MHBauer) [inline pkg/apiserver/apiserver.go]: imports okay?

**Comment by MHBauer** [comment]
@duglin @martinmaly resolved?

File: pkg/apiserver/apiserver.go (hunk 1 above)
Comment (duglin) [inline pkg/apiserver/apiserver.go]: we'll get it later

**Comment by duglin** [comment]
LGTM

**Pull request merged**